packages/fluentui/react-icons-northstar/src/index.ts

```diff
@@ -162,11 +169,14 @@ export { default as SendIcon } from './components/SendIcon';
 export { default as SettingsIcon } from './components/SettingsIcon';
 export { default as SettingsAudioIcon } from './components/SettingsAudioIcon';
 export { default as ShareAltIcon } from './components/ShareAltIcon';
+export { default as ShareGenericIcon } from './components/ShareGenericIcon';
 export { default as ShareLocationIcon } from './components/ShareLocationIcon';
+export { default as ShareLocation12pxIcon } from './components/ShareLocation12pxIcon';
```

Comment:
Can we change the name a bit and avoid using `12px`?

Reply:
Could we please not? I don't think using "small", "smaller", "verysmall" etc names is better

Reply:
We have `size` prop for that. Why are we not rendering this as a dedicated svg for some size then?

Reply:
Because it is designed explicitly to be used at 12px.

Reply:
My point was that `<ShareLocationIcon size="small" />` for example can render that svg. I don't like `12px` in the name, because `<ShareLocation12pxIcon size="large" />` will definitelly not be 12px..

Reply:
But it's not meant to be used as "large". We already have icons that have this in the name, how else a dev or anyone else should know that this is very-very specific icon. Besides - I was asked to modify existing icon which already has this kind of name. Included in this PR